src/atoms/sdp_cone/lambda_min_max.jl

```diff
@@ -39,7 +39,8 @@ function curvature(x::LambdaMaxAtom)
 end
 
 function evaluate(x::LambdaMaxAtom)
-    eigvals(evaluate(x.children[1]))[end]
+    #eigvals(evaluate(x.children[1]))[end]
+    sort(eigvals(evaluate(x.children[1])), by = x -> real(x))[end]
```

Comment:
```suggestion
    sort!(eigvals(evaluate(x.children[1])), by=real)[end]
```
Same below. Should also delete the line above instead of leaving it commented out.